### PR TITLE
[-] FO :  Access address

### PR DESCRIPTION
**Pull request by Shine-neko**
Forge Ticket:

http://forge.prestashop.com/browse/BOOM-437

### CODE REVIEW

File: classes/form/CustomerAddressForm.php [hunk 1]
```diff
@@ -42,6 +42,7 @@ public function __construct(
     public function loadAddressById($id_address)
     {
         $address = new Address($id_address, $this->language->id);
```
Comment (mickaelandrieu) [inline classes/form/CustomerAddressForm.php]: how about ``$this->address = new Address($id_address, $this->language->id); `` ?

File: controllers/front/AddressController.php [hunk 2]
```diff
@@ -78,7 +89,7 @@ public function postProcess()
                     $this->errors[] = $this->l('Could not delete address.');
                 }
             } else {
-                $this->address_form->loadAddressById($id_address);
+                ;
```
Comment (mickaelandrieu) [inline controllers/front/AddressController.php]: I'm 100% sure this line will fails :cat:

Comment (Shine-neko) [inline controllers/front/AddressController.php]: Nop :dancer:

Comment (mickaelandrieu) [inline controllers/front/AddressController.php]: ^_^ anyway, remove it !

**Comment by mickaelandrieu** [comment]
Thank you @Shine-neko

**Pull request merged**